beluga/test/beluga/CMakeLists.txt

```diff
@@ -52,7 +53,6 @@ add_executable(
   sensor/test_beam_model.cpp
   sensor/test_bearing_sensor_model.cpp
   sensor/test_landmark_sensor_model.cpp
-  sensor/test_lfm_with_unknown_space.cpp
```

Comment:
@pvela2017 why was this removed?

Reply:
Mistakes, added again in f1e7b760dae883db0c95f9584e06d82f35e5b1de